clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/ops/UpsertGoogleAutoscalingPolicyAtomicOperation.groovy

```diff
@@ -68,6 +69,12 @@ class UpsertGoogleAutoscalingPolicyAtomicOperation extends GoogleAtomicOperation
     this.description = description
   }
 
+  UpsertGoogleAutoscalingPolicyAtomicOperation(UpsertGoogleAutoscalingPolicyDescription description, GoogleClusterProvider googleClusterProvider, GoogleOperationPoller googleOperationPoller) {
```

Comment:
Let's try for one constructor that takes all currently autowired members, and description, so the `@Autowired` annotation disappears for all members.

Reply:
It'll be `refactor(google)` since the change isn't limited to tests.

Reply:
Updated the code.